packages/wdio-protocols/protocols/webdriver.json

```diff
@@ -679,7 +679,14 @@
         "name": "propertyName",
         "description": "name of the CSS property to retrieve"
       }],
-      "parameters": [],
+      "parameters": [
+        {
+          "type": "String|Null",
+          "required": false,
+          "name": "pseudoElement",
+          "description": "the pseudo-element to match. Omitted (or null) for real elements"
+        }
+      ],
```

Comment:
Can you provide a reference where this is defined within the [WebDriver spec](https://w3c.github.io/webdriver/webdriver-spec.html)?

Reply:
Do I add it in the description, ref doesn't seem to work?
```diff
- "description": "the pseudo-element to match. Omitted (or null) for real elements" 
+ "description": "the pseudo-element to match. Omitted (or null) for real elements. See: https://drafts.csswg.org/cssom/#dom-window-getcomputedstyle"
```

Reply:
That is not what I meant here. You are defining a command for the WebDriver spec. However the spec has to ability to fetch pseudo elements. WebdriverIO generates its command set from these information and if someone would use this parameter it would not work.